pkg/auth/migrate/rbac_to_acl.go

```diff
@@ -114,6 +118,35 @@ func RBACToACL(ctx context.Context, svc auth.Service, doUpdate bool, creationTim
 		messageFunc(group.DisplayName, *newACL, warnings)
 	}
 
+	users, _, err := svc.ListUsers(ctx, &model.PaginationParams{Amount: maxUsers + 1})
+	if err != nil {
+		return fmt.Errorf("list users: %w", err)
+	}
+	if len(users) > maxUsers {
+		return fmt.Errorf("%w (got %d)", ErrTooManyUsers, len(users))
+	}
+	for _, user := range users {
+		var warnings error
```

Comment:
Doesn't this cause all earnings here to be appended to the wrong var and lost to the caller?

Reply:
+1

Reply:
@arielshaqed , @ortz Why? warnings are used within this closer, messageFunc(user.Username, model.ACL{}, warnings), and it's erased every iteration, same as the logic in groups.
What am I missing?

Reply:
Anyway I changed the implementation, not using warning anymore for users

Reply:
But shadowing warnings over here means these warnings are always lost